package.json

```diff
@@ -13,8 +13,8 @@
     "prepare": "patch-package",
     "build:tsc": "tsc --outDir build",
     "build": "npm-run-all -s generate:api-models build:tsc",
-    "generate:api-models:public": "gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/master/api/public_api_v1.yaml --no-strict --out-dir src/api",
-    "generate:api-models:admin": "gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/master/api/admin_api.yaml --no-strict --out-dir src/api",
+    "generate:api-models:public": "rimraf src/api && mkdir -p src/api && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/master/api/public_api_v1.yaml --no-strict --out-dir src/api",
```

Comment:
I suggest to keep the generated source code separate from the rest, in other projects we put everything under `generated` that is a sibling to `src`